src/providers/twitch/TwitchChannel.hpp

```diff
@@ -92,6 +92,12 @@ class TwitchChannel : public Channel, pajlada::Signals::SignalHolder
     boost::optional<EmotePtr> twitchBadge(const QString &set,
                                           const QString &version) const;
 
+    // Cheers
+    boost::optional<
+        std::tuple<boost::optional<EmotePtr>, boost::optional<EmotePtr>,
+                   boost::optional<QColor>>>
```

Comment:
Could just use [CheerEmote](https://github.com/Chatterino/chatterino2/blob/master/src/providers/twitch/TwitchEmotes.hpp#L18) instead of a tuple 😄